core/storage/email/gae_models.py

```diff
@@ -562,5 +581,7 @@ def export_data(cls, user_id):
         user_data = {}
         email_reply_models = cls.get_all().filter(cls.user_id == user_id)
         for model in email_reply_models:
-            user_data[model.thread_id] = model.reply_to_id
+            user_data[model.thread_id] = {
+                'reply_to_id': model.reply_to_id
```

Comment:
I think I'm a bit confused about how this works. How does the general verification checker know that it needs to read the model.thread_id property to get the reply_to_id? I don't see any export-related constants that encode this structure.

Also, unrelatedly, please change GeneralFeedbackEmailReplyToIdModel's in the docstring for this method to FeedbackEmailReplyToIdModel's.

Reply:
Discussed offline, the verification checker iterates over the keys and then verifies according to the export policy.

Reply:
Addressed the docstring, thanks for catching that. Also, yes the testing iterates over the keys in the export and verifies that they match the export policy.